lldb/cmake/modules/AddLLDB.cmake

```diff
@@ -182,6 +182,13 @@ function(add_properties_for_swift_modules target)
 
     set_property(TARGET ${target} APPEND PROPERTY BUILD_RPATH "${SWIFT_RPATH}")
     set_property(TARGET ${target} APPEND PROPERTY INSTALL_RPATH "${SWIFT_RPATH}")
+
+    if (SWIFT_SWIFT_PARSER)
+      set_property(TARGET ${target}
```

Comment:
Should we add those paths only if we are building for Darwin (maybe by using a generator expression like `$<$<BOOL:${LLVM_LINKER_IS_LD64}>:...>`, or would those work for other platforms as well?

Reply:
We use `@loader_path` in a lot of places that don't see Darwin-specific already. Perhaps it's getting filtered at some point later?